### DISCUSSION TITLE
Filament 404 Not Found Error After Login Successfully.

### DISCUSSION THREAD

**Discussion by mayuriflair** [filamentphp/filament#14135] [Help]
### Package

Panel builder

### Package Version

v3.2.110

### How can we help you?

When I try to log in using the admin credentials, it shows a 404 not found popup after sending the correct email and password. However, the user session token is generated by the system automatically and stored in the database so when refresh the page after having a 404 not found error, it redirects me to the dashboard page. Please let me know if any knows how to resolve this kind of error.

Thanks.

Ref: https://github.com/user-attachments/assets/fc631e9d-bf84-44bb-bf6f-1677a4e9cf5d

**Comment by willcastillo** [comment]
Getting the same error. In my case, the 404 is produced on [this line](https://github.com/filamentphp/filament/blob/3.x/packages/panels/src/Pages/Auth/Login.php#L63)

**Comment by willcastillo** [comment]
Just to be clear: that's exactly the line that is throwing the error, **not** the next one (`$this->throwFailureValidationException();`)

**Reply by leandrocfe** [reply]
Any Filament customizations?
Did you try this in a Filament fresh install?

**Reply by mayuriflair** [reply]
@leandrocfe Without adding any customization in the Filament, this error persists from the same as mentioned by @willcastillo 

I didn't apply any customization in the filament and tried to log into Filament Dashboard but it shows a 404 Not Found error.

**Reply by leandrocfe** [reply]
Are you able to share this repo on github?

**Reply by mayuriflair** [reply]
@leandrocfe Sorry, I can't share the repo on GitHub with you because of some restrictions.

**Reply by leandrocfe** [reply]
It's difficult to assist without any code. Maybe you could create a minimal repo on the Github to reproduce this issue.

**Reply by willcastillo** [reply]
Hey folks, fwiw the issue on my end "fix itself" after running a composer update, unfortunately I did not take note of the previous version I was using :(

**Reply by mayuriflair** [reply]
@leandrocfe and @willcastillo I have resolved this issue so we can close this discussion.

**Comment by mayuriflair** [comment]
There was an issue with the external package that was installed within the Filament project and I have resolved the issue by using Laravel Debugger that helped me to track and resolve the issue.

**Reply by ruwad-co** [reply]
You are the best thank you
